test/feeder/api_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Feeder.ApiTest do
+  use Castle.HttpCase
+
+  @all "per=100&since=1970-01-01"
+  @podcasts "https://feeder.prx.org/api/v1/podcasts"
+  @episodes "https://feeder.prx.org/api/v1/episodes"
+  @all_podcasts "#{@podcasts}?#{@all}"
+
+  test_with_http "handles 404s", %{@all_podcasts => 404} do
```

Comment:
Some fun metaprogramming.  In `HttpCase`, it's actually using `Mock` to mock that hash of http calls (similar to nock in nodejs), then executes the block.